simplic-user/Simplic.User.UI/UserControls/MultiSelectComboBox.xaml

```diff
@@ -0,0 +1,83 @@
+﻿<UserControl x:Class="Simplic.User.UI.MultiSelectComboBox"
```

Comment:
Such "base" controls should never be implemented in a library as "User.UI", since this is more general and might me useful for more. Because if we want to make changes, we need to make them in all projects.

Reply:
Implementations of these user controls are different from project to project. These controls are different by properties and by inner logic, which can't extract for creating more general control.